composer.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "symplify/symplify",
     "description": "Main repository for Symplify packages development.",
+    "license": "MIT",
```

Comment:
I was not only one who noticed - as that was the point of this experiment.

Packagist is quite loud about it: https://github.com/nette/coding-standard/pull/2#issuecomment-301596140